### PR TITLE
WIP v0.6 license and markdown

### PR DESCRIPTION
**Pull request by dawnwages**
Work in progress pull request
https://docs.google.com/document/d/1lMrHHczeigA-u9C1rWkZuiPXV0Q7GTkX0GXDOefltbA/edit

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+   * [3.1.15.](#3.1.15)  Arbitrarily deprive any person of his/her/their property;
+   * [3.1.16.](#3.1.16)  Forcibly remove indigenous peoples from their lands or territories or take any action with the aim or effect of dispossessing indigenous peoples from their lands, territories, or resources, including without limitation the intellectual property or traditional knowledge of indigenous peoples, without the free, prior, and informed consent of indigenous peoples concerned;
+   * [3.1.17.](#3.1.15) Perpetuate inequality in healthcare services for people on the basis of race, ethnicity, nationality, caste, phenotype or genotype, including, but not limited to:  
+     * [3.1.17.1.](#3.1.17.1)  Claiming medical devices, treatments, or procedures, which are untested or tested and known to be harmful,  as sufficient and exhaustive medical care without express written consent by the patient, or someone with authority to consent on behalf of the patient, acknowledging the unknown or harmful properties of the medical device, treatment, or procedure, and authorizing its use;
```
Comment (LeeGee64) [inline LICENSE:72]: "Claiming" needs to be "claim" in order to fit with the grammar of the section

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+     * [3.1.17.2.](#3.1.17.2)  Using race-adjusted or “race-norming” formulas that put patients at undue risk or result in insufficient standard of care;
```
Comment (LeeGee64) [inline LICENSE:73]: "Using"  becomes "use" as per line 72

File: LICENSE [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+     * [3.1.17.3.](#3.1.17.3)   Profiling patients solely on the basis of race, ethnicity, nationality, caste, phenotype or genotype, or in any other manner which would lead to care which is either negligent or  insufficient to meet the standard of care;
```
Comment (LeeGee64) [inline LICENSE:74]: "Profiling" --> "Profile"

**Review by LeeGee64 — COMMENTED** [review]
Left a few grammar comments here and there but it looks good!

File: LICENSE (hunk 1 above)
Comment (dawnwages) [reply LICENSE:72]: this section would be "including, but not limited to claiming medical devices...", "including, but not limited to using race-adjusted...", "including but not limited to, profiling..." 

I think this one works, but it is odd. I also see an extra space in there. I'll remove that.

**Review by gtothe7 — COMMENTED** [review]
LICENSE

**Pull request merged**